packages/swift-sdk/get_logs.sh

```
@@ -0,0 +1,238 @@
+  sessions+=("$s")
+done < <(find "$root" -mindepth 1 -maxdepth 1 -type d \
+  -name '[0-9]*' -exec basename {} \; | sort -r)
+
+if [ "${#sessions[@]}" -eq 0 ]; then
+  echo "No session directories under $root." >&2
+  echo "Check that the app actually called LoggingPreferences.configure()" >&2
+  exit 1
+fi
+
+pick_session() {
+  if [ "$SESSION" = "latest" ] || [ -z "$SESSION" -a "${#sessions[@]}" -eq 1 ]; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)